### PR TITLE
added constructor to gameInstance.java

### PR DESCRIPTION
**Pull request by tch1001**
this is my first github change!!! pls forgive me if i did anything wrong
i forgot the keyword 'int' in constructor...

### CODE REVIEW

File: src/main/java/net/towerwarz/GameInstance.java [hunk 1]
```diff
@@ -33,6 +33,17 @@
             "Sparta"
     };
 
+    public GameInstance(String difficulty, mapName){
```
Comment (tch1001) [inline src/main/java/net/towerwarz/GameInstance.java]: forgot int for mapName sorry

**Comment by bushshrub** [comment]
The continuous integration checks failed. Could you fix the syntax errors

**Comment by xiurobert** [comment]
I'll check once I get back

**Comment by xiurobert** [comment]
@lightningsf-ci test this please

**Comment by bushshrub** [comment]
@lightningsf-ci test this please

**Comment by bushshrub** [comment]
@lightningsf-ci add to whitelist

**Pull request merged**